modules/rte_mis_mail/src/Form/SMSSettingsForm.php

```diff
@@ -0,0 +1,118 @@
+<?php
```

Comment:
Let's change the file name to `SmsSettingsForm`

Reply:
modified to email_sms_form.settings